pkg/installer/installer.go

```diff
@@ -126,19 +126,63 @@ func Purge() {
 	purge(defaultLocksPath, defaultRepositoriesPath)
 }
 
+// PurgePackage removes an individual package
+func PurgePackage(pkg string) {
+	purgePackage(pkg, defaultLocksPath, defaultRepositoriesPath)
+}
+
+func purgePackage(pkg string, locksPath, repositoryPath string) {
+	var err error
+	span, ctx := tracer.StartSpanFromContext(context.Background(), "purge_pkg")
+	defer span.Finish(tracer.WithError(err))
+	span.SetTag("params.pkg", pkg)
+
+	switch pkg {
+	case "datadog-agent":
+		service.RemoveAgentUnits(ctx)
+		if err = removePkgDirs(ctx, pkg, locksPath, repositoryPath); err != nil {
+			log.Warnf("installer: %v", err)
+		}
+	case "datadog-apm-inject":
+		if err = service.RemoveAPMInjector(ctx); err != nil {
+			log.Warnf("installer: could not remove APM injector: %v", err)
+		}
+		if err = removePkgDirs(ctx, pkg, locksPath, repositoryPath); err != nil {
+			log.Warnf("installer: %v", err)
+		}
+	default:
+		log.Warnf("installer: unrecognized package purge")
+	}
+}
+
 func purge(locksPath, repositoryPath string) {
 	var err error
 	span, ctx := tracer.StartSpanFromContext(context.Background(), "purge")
 	defer span.Finish(tracer.WithError(err))
-	service.RemoveAgentUnits(ctx)
```

Comment:
can you remove also the APMInjector purge to avoid impacting injection
+ add a TODO to properly support them

Reply:
The APM injection removal already checks that the LD_PRELOAD points to the injector in `/opt/datadog-packages` before removal, so it should be fine to leave it.

But since for now there is no point in taking this codepath since customers won't install injectors, I'll comment it out